packages/babel-make-styles/src/plugin.ts

```diff
@@ -448,30 +461,31 @@ export const plugin = declare<never, PluginObj<BabelPluginState>>(api => {
                 );
               }
 
-              const stylesBySlots: Record<string, MakeStyles> = evaluationResult.value;
-              const resolvedStyles: ResolvedStylesBySlots<string> = {};
-
-              Object.keys(stylesBySlots).forEach(slotName => {
-                resolvedStyles[slotName] = resolveStyleRules(stylesBySlots[slotName]);
-              });
-
-              nodePath.replaceWithMultiple((astify(resolvedStyles) as t.ObjectExpression).properties);
+              const stylesBySlots: StylesBySlots<string, unknown> = evaluationResult.value;
 
-              return;
+              nodePath.replaceWithMultiple((astify(stylesBySlots) as t.ObjectExpression).properties);
             }
+          });
+
+          state.calleePaths?.forEach(calleePath => {
+            const callExpressionPath = calleePath.findParent(parentPath =>
+              parentPath.isCallExpression(),
+            ) as NodePath<t.CallExpression>;
+            const argumentPath = callExpressionPath.get('arguments.0') as NodePath<t.ObjectExpression>;
 
-            const evaluationResult = nodePath.evaluate();
+            const evaluationResult = argumentPath.evaluate();
 
             if (!evaluationResult.confident) {
-              throw nodePath.buildCodeFrameError(
+              throw argumentPath.buildCodeFrameError(
                 'Evaluation of a code fragment failed, this is a bug, please report it',
               );
             }
 
-            const styles: MakeStyles = evaluationResult.value;
-            const resolvedStyles = resolveStyleRules(styles);
+            const stylesBySlots: StylesBySlots<string, unknown> = evaluationResult.value;
```

Comment:
why `unknown` ? it's a function that returns an object or and object right ?

Reply:
The type was wrong there, is should an object of objects (`Record<string /* slot */, object /* styles */>`). Fixed 👍